src/style.css

```diff
@@ -1,3 +1,8 @@
+.app-body{
```

Comment:
there will be only one body tag in your HTML so can use css HTML tag selector, directly here.
body{
}